aws/aws_instance.go

```diff
@@ -218,20 +218,20 @@ func (p *AWS) CreateInstance(ctx *lepton.Context) error {
 	var sg *ec2.SecurityGroup
 
 	if cloudConfig.SecurityGroup != "" && cloudConfig.VPC != "" {
-		ctx.Logger().Debug("getting security group with name %s", cloudConfig.SecurityGroup)
+		ctx.Logger().Debugf("getting security group with name %s", cloudConfig.SecurityGroup)
 		sg, err = p.GetSecurityGroup(ctx, svc, vpc)
 		if err != nil {
 			return err
 		}
 	} else {
-		ctx.Logger().Debug("creating new security group in vpc %s", *vpc.VpcId)
+		ctx.Logger().Debugf("creating new security group in vpc %s", *vpc.VpcId)
 		sg, err = p.CreateSG(ctx, svc, imgName, *vpc.VpcId)
 		if err != nil {
 			return err
 		}
 	}
 
-	ctx.Logger().Debug("getting subnet")
+	ctx.Logger().Debugf("getting subnet")
```

Comment:
not formatted?

Reply:
Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73